harness/tests/launch/test_deepspeed.py

```diff
@@ -266,3 +266,24 @@ def test_launch_worker(
 
     expected_cmd = pid_server_cmd + sshd_cmd
     mock_subprocess.assert_called_once_with(expected_cmd)
+
+
+def test_filter_env_vars() -> None:
```

Comment:
nits: could we add some of the `DET_xxx` env vars that users might want to `keep`? Like `DET_USER_TOKEN`,  `DET_SESSION_TOKEN`, etc.

I think the test would also be a little more readable if we had like `env_exp = {k:v for k,v in env_in.items() if v == "keep"}`, to avoid needing to cross-reference.

Reply:
sure